libraries/Microsoft.Bot.Schema/Teams/MeetingTabIconSurface.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Schema.Teams
+{
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Specifies meeting stage surface.
+    /// </summary>
+    public class MeetingTabIconSurface : Surface
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MeetingTabIconSurface"/> class.
+        /// </summary>
+        public MeetingTabIconSurface()
+            : base(SurfaceType.MeetingTabIcon)
+        {
+        }
+
+        /// <summary>
+        /// Gets or sets the tab entity Id of this <see cref="MeetingTabIconSurface"/>.
+        /// </summary>
+        /// <value>
+        /// The tab entity Id of this <see cref="MeetingTabIconSurface"/>.
+        /// </value>
+        [JsonProperty("tabEntityId")]
+        public string TabEntityId { get; set; }
```

Comment:
As this can be an optional parameter and it is public facing code, can we mark it as nullable type to make the intention explicit?

Reply:
Updated in the comment this is an optional field. 
Due to C# version SDK is using, string not support nullable type.